### ISSUE TITLE
Authorizer not setting Header

### ISSUE THREAD

**Issue by wieseljonas** [fenichelar/ember-simple-auth-token#6]
Hi I've tried to make the recent versions work. Seems like I can get the authentication header to be added to the requests.

Any ideas why it's not working?

here is my index.html

```
 window.ENV['simple-auth'] = {
    authorizer: 'authorizer:token'
  };
  window.ENV['simple-auth-token'] = {
    serverTokenEndpoint: 'http://localhost:1337/api/v1/login',
    authorizationPrefix: '',
    tokenPropertyName: 'token',
    authorizationHeaderName: 'access_token',
    identificationField: 'username'
  };
```

My route

```
import Ember from 'ember';
import AuthenticatedRouteMixin from 'simple-auth/mixins/authenticated-route-mixin';

export default Ember.Route.extend(AuthenticatedRouteMixin, {
model: function() {
    return this.store.find('testrestrictedmodel');
}
});
```

**Comment by marcemira** [comment]
Same here... :/

**Comment by jpadilla** [comment]
Hey thanks for reporting and sorry for the trouble. I'll look into this today. I'm in the process of rewriting how this is packaged using `ember addon <addon-name>` and getting some tests done.

**Comment by marcemira** [comment]
:+1:  Gracias josé!

**Comment by jpadilla** [comment]
@wieseljonas @marcemira A couple of questions to help me debug this. I just created a project from scratch and got it working.
- Are both of you using ember-data?
- Is the API you're connecting to on the same domain/ip? Wondering if it's a `crossOriginWhitelist` issue.

**Comment by wieseljonas** [comment]
I'll give it as spin and get back to you @jpadilla

**Comment by rbartoli** [comment]
@jpadilla While building my app I've encountered this error.

My API is on http://localhost:3000 while the page is served from http://0.0.0.0:4200. I'm also using ember-data.

Adding the API IP address in the `crossOriginWhitelist` configuration property fixed the issue.

**Comment by jpadilla** [comment]
@rbartoli thanks for confirming

**Comment by jpadilla** [comment]
@wieseljonas @marcemira @rbartoli  I've been working into refactoring this to use the new ember-cli's `ember addon` feature as well as adding tests.

I've created a new branch, [refactor](https://github.com/jpadilla/ember-cli-simple-auth-token/tree/refactor) with the current progress which seems to be working on an example project. All tests are passing.

I'd like some feedback on it and if you could try it out and report any weird things, that'd be awesome.

**Comment by jpadilla** [comment]
Just released [v0.4.0](https://github.com/jpadilla/ember-cli-simple-auth-token/releases/tag/v0.4.0).

**Comment by alekspetrov** [comment]
Header is not present in request. 

```
ENV['simple-auth'] = {
    crossOriginWhitelist: ['*'],
    authorizer: 'simple-auth-authorizer:token',
    authenticationRoute: '/',
    routeAfterAuthentication: '/photos',
    store: 'simple-auth-session-store:local-storage'
  };
```

```

  ENV['simple-auth-token'] = {
    serverTokenEndpoint: 'http://my-host/login',
    identificationField: 'identification',
    passwordField: 'password',
    tokenPropertyName: 'token',
    authorizationPrefix: 'Bearer ',
    authorizationHeaderName: 'Authorization',
    headers: {}
  };
```

Backend response  `{"token":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE0MzcxOTk5NDR9.CXk7Adw19d43dWq0PsisAg1YQilxYMeIgif-rm0rEv0"}`
